runtime/doc/options.txt

```diff
@@ -3803,7 +3802,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	first character in a pair gives the occasion, the second the mode to
 	use for that occasion.  The occasions are:
 	|hl-SpecialKey|	 8  Meta and special keys listed with ":map"
-	|hl-NonText|	 @  '~' and '@' at the end of the window and
+	|hl-EndOfBuffer|      ~ lines after the last line in the buffer
+	|hl-NonText|	 @  '@' at the end of the window and
```

Comment:
I think this needs a `{Nvim}` and/or a mention at https://github.com/neovim/neovim/wiki/Differences-from-Vim. I fear that we'll get to a point where we lose track of what small feature changes such as this Neovim has compared to Vim. Maybe we're already there...